components/sections/ProjectInProgress.tsx

```diff
@@ -1,12 +1,13 @@
-import {Card, ProjectInProgressData} from 'project-reports/project-in-progress'
-import {useMemo} from 'react'
-import {CellProps, Column} from 'react-table'
-import {getStatusEmoji} from '../../lib/util'
-import CardAssignee, {getAssignee} from '../CardAssignee'
+import { Card, ProjectInProgressData } from 'project-reports/project-in-progress'
```

Comment:
We have Prettier.js and a configuration for it set up in the repo—if you run `npm run check:format` it should format everything for you automatically in line with the rest of the repo. There should be a pre-commit hook doing this, but it doesn't appear that it ran here for some reason.